lib/hammer_cli_katello/repository.rb

```diff
@@ -465,10 +465,12 @@ def update_content_upload(upload_id, repo_id, file)
       def import_uploads(uploads, opts = {})
         publish_repository = opts.fetch(:last_file, false)
         sync_capsule = opts.fetch(:last_file, false)
+        async = options["option_content_type"] == 'ostree_ref'
```

Comment:
I think all types should use the async option IMO.  There are other ways for the task to take too long (such as if all pulp workers are busy syncing repos).   That might make it easier below if we want this command to poll tasks always